content/src/logic/deployments.ts

```diff
@@ -44,6 +51,10 @@ export async function retryFailedDeploymentExecution(
           DeploymentContext.FIX_ATTEMPT
         )
       } catch (error) {
+        // it failed again, override failed deployment error description
```

Comment:
Are you sure that if the deployment fails throws an exception? Can it fail without throwing an error?